invoice.go

```diff
@@ -119,6 +119,7 @@ type InvoiceParams struct {
 	SubscriptionItems                       []*SubscriptionItemsParams          `form:"subscription_items"`
 	SubscriptionPlan                        *string                             `form:"subscription_plan"`
 	SubscriptionProrate                     *bool                               `form:"subscription_prorate"`
+	SubscriptionProrationBehavior           *string                             `form:"subscription_proration_behavior"`
```

Comment:
You're missing a list of constants for all those values. In go, we define them even if it's just used as a parameter and we pass `*string`

Reply:
Thanks, I couldn't remember that rule.